tests/foreman/performance/common/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
+
+Part of functionalities of Candlepin are defined in this module
+and have utilities of single register by activation-key, single
+register and attach, single subscription deletion.
+
+"""
+
+
+from robottelo.common import conf, ssh
+
+import logging
+import time
+import requests
+
+
+class CandlepinPerfTest:
+    """Measures performance of RH Satellite 6
+
+    Candlepin Subscription functionality
+    """
+    username = conf.properties['foreman.admin.username']
+    password = conf.properties['foreman.admin.password']
+    serverhost = conf.properties['main.server.hostname']
+
+    # create logger
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
+    requests.packages.urllib3.disable_warnings()
```

Comment:
I think is better to run this after https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/__init__.py#L39. This will make the warning being disabled for API tests too.